src/lib/comparison.js

```diff
@@ -67,6 +67,20 @@ function compareIn(field, values, normaliseValues) {
   return joinRules(compareMap(field, 'eq', values, normaliseValues), 'or');
 }
 
+function compareAll(objectValue, normaliseValues) {
+  const keys = Object.keys(objectValue);
+  return joinRules(keys.map(k => {
+    const ov = objectValue[k];
+
+    if(Array.isArray(ov)) {
+      return `(${compareIn(k, ov, normaliseValues)})`;
+    } else {
```

Comment:
What do you think about how to handle undefined values?
```
{
  name: 'john doe',
  age: undefined
}
```
It relevant to how should we handle `filterBuilder.eq('age', undefined)`(#18)

Reply:
@bodia-uz  - I filter out the undefined values (silently skipped the properties)